api/about.json

```diff
@@ -3,7 +3,7 @@
    {
     "logo": "http://bit.ly/2niBg5Y",
     "website":"https://pydelhi.org/",
-    "details":"PyDelhi is a volunteer organization, group of technically inclined students, professionals, teachers and entrepreneurs who want to learn something new having a common primary interest in Python programming language." 
+    "about":"PyDelhi Conf is the annual conference hosted by PyDelhi community.It will be bringing together developers, experts, researchers, scientists, enthusiasts and companies dealing in python.It is a completely volunteer based event, managed by PyDelhi(https://pydelhi.org/) python users group Delhi.On its maiden year PyDelhi Conf attracted a total attendance of 300 people." 
```

Comment:
@shivan1b Alright, sounds good. Please ensure that they are properly updated on website as well as app. 

Tagging @realslimshanky to ensure that this goes good on website, if needed.